hypertrace-core-graphql-span-schema/src/main/java/org/hypertrace/core/graphql/span/dao/SpanDaoModule.java

```diff
@@ -27,11 +31,15 @@
   @Override
   protected void configure() {
     bind(SpanDao.class).to(GatewayServiceSpanDao.class);
-
+    bind(GatewayServiceFutureStub.class)
```

Comment:
If we're going to make this a common injectable (I generally just share the channels and recreate the stubs, but have no issues with this) it should be owned by a common module - probably `GatewayUtilsModule`

Reply:
How about `GraphQlGrpcModule` ?

Reply:
That module doesn't have the context of gateway service - gatewayutils does (and gateway implies grpc since that's how it's implemented)

Reply:
yeah realised that, adding it in `GatewayUtilsModule`